.changeset/great-lions-buy.md

```diff
@@ -0,0 +1,9 @@
+---
+'@xstate/test': major
+---
+
+@author: @mattpocock @davidkpiano
```

Comment:
This won't work correctly (although I think that it would be nice if it could):
https://github.com/changesets/changesets/blob/34933f041ab87d0fd1dc0250b894e011cf12da00/packages/changelog-github/src/index.ts#L64-L67

```suggestion
@author: @mattpocock 
@author: @davidkpiano
```